Update impact description for protocol range changes (#5191)
SkylineCommunications/dataminer-docs: develop/devguide/Connector/ImpactOfProtocolChanges/ImpactOfProtocolVersionChanges.md

There are many protocol changes that can cause impact.

> The nature and severity of the impact will be documented via the connector XML [VersionHistory](xref:Manifest.VersionHistory) subtags as well as the actions to take to overcome those. This information will then be exposed both in the Catalog and in DataMiner Cube when a pre-existing element is updated.

Reviewer: Something isn't quite right in this sentence. Do I interpret everything correctly if I change it like this?
```suggestion
The nature and severity of the impact will be documented via the connector XML [VersionHistory](xref:Manifest.VersionHistory) subtags along with the actions that need to be taken to overcome the impact. This information will then be exposed both in the Catalog and in DataMiner Cube when a pre-existing element is updated.
```

Author: correct!
